relnotes/taskpool.feature.md

```diff
@@ -0,0 +1,4 @@
+* `ocean.task.TaskPool`
+
+  A new function `awaitAllTasks()` has been added. This function suspends the
+  current task until all tasks in the pool have finished executing.
```

Comment:
Not having looked at the code, I find myself wondering at this point "all tasks including the task that's being suspended?" (= impossible).